README.md

```diff
@@ -10,134 +10,68 @@ This is used by [Liftoff](http://github.com/tkellen/node-liftoff) to automatical
 
 ### extensions
 Map file types to modules which provide a [require.extensions] loader.
+
 ```js
 {
-  '.babel.js': 'babel/register',
+  '.babel.js': {
+    module: 'babel/register',
+    register: function (module) {
+      module({
+        // register on .js extension due to https://github.com/joyent/node/blob/v0.12.0/lib/module.js#L353
+        // which only captures the final extension (.babel.js -> .js)
+        extensions: '.js'
+      })
+    }
+  }
   '.cirru': 'cirru-script/lib/register',
   '.cjsx': 'node-cjsx/register',
   '.co': 'coco',
-  '.coffee': 'coffee-script/register',
-  '.coffee.md': 'coffee-script/register',
+  '.coffee': ['coffee-script/register', 'coffee-script'],
+  '.coffee.md': ['coffee-script/register', 'coffee-script'],
   '.csv': 'require-csv',
-  '.iced': 'iced-coffee-script/register',
-  '.iced.md': 'iced-coffee-script/register',
+  '.iced': ['iced-coffee-script/register', 'iced-coffee-script'],
+  '.iced.md': ['iced-coffee-script/register', 'iced-coffee-script'],
   '.ini': 'require-ini',
   '.js': null,
   '.json': null,
   '.json5': 'json5/lib/require',
-  '.jsx': 'node-jsx',
-  '.litcoffee': 'coffee-script/register',
-  '.liticed': 'iced-coffee-script/register',
-  '.ls': 'LiveScript',
+  '.jsx': [
```

Comment:
i mean, have the first entry be babel with a different register function (register on .jsx instead of .js) and then have node-jsx as the fallback